src/audio/dai.c

```diff
@@ -53,7 +53,7 @@ struct dai_data {
 	uint32_t stream_id;
 	struct dma_sg_config config;
 	struct comp_buffer *dma_buffer;
-
+	struct sof_timestamp_cfg ts_config;
```

Comment:
nitpick. no need for the sof_ prefix since this is only used internally and not in ABI.

Reply:
I forgot to do this, will do for next version!

Reply:
Also "struct sof_timestamp_data" will be renamed, it's similar.

Reply:
Done!